parser/parser.py

```diff
@@ -65,7 +65,8 @@ def process_regions(root_path):
 
         # if link starts with ../ create a dita file for the country => example ../Britain/Britain1.html
         link = area["href"]
-        country_name = area["alt"]
+        #country_name = area["alt"]
+        country_name = os.path.dirname(link[3:])
```

Comment:
in the next `if` block we test if the link starts with `../`.  If that's true, then it's ok to trim the name like you have here.  But if the link doesn't start with `../` then it could just mangle the real name.